Doc/library/sys.rst

```diff
@@ -1368,13 +1368,26 @@ always available.
    returned by the :func:`open` function.  Their parameters are chosen as
    follows:
 
-   * The character encoding is platform-dependent.  Under Windows, if the stream
-     is interactive (that is, if its :meth:`isatty` method returns ``True``), the
-     console codepage is used, otherwise the ANSI code page.  Under other
-     platforms, the locale encoding is used (see :meth:`locale.getpreferredencoding`).
-
-     Under all platforms though, you can override this value by setting the
-     :envvar:`PYTHONIOENCODING` environment variable before starting Python.
+   * The character encoding is platform-dependent.  Non-Windows
+     platforms use the locale encoding (see
+     :meth:`locale.getpreferredencoding()`).
+     On Windows, UTF-8 is used for console character
+     devices (i.e. CON, CONIN$, and CONOUT$).  However, this
+     can be overridden to use the console as a generic
+     character device by setting the environment variable
+     :envvar:`PYTHONLEGACYWINDOWSSTDIO` before starting Python.  Non-
+     character devices such as disk files and pipes use the
```

Comment:
Any ReST experts (or someone with a build of this) know whether hyphens are handled properly across line breaks? Will this end up as "Non-character" or "Non- character"?

Reply:
Pretty sure line breaks turn into spaces, just like in HTML.

Reply:
According to my local Docs build, it is indeed rendered as "Non- character". 
It should now be alright.